### PR TITLE
[plugin.video.regiotv] v0.1.0

### PR DESCRIPTION
**Pull request by dagwieers**
### Description
This is a new add-on for (Flemish) regional TV stations.

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by basrieter** [comment]
Would it make sense to have it called `plugin.video.regiotvbe` to perhaps not interfere with future other regio tv's?

**Comment by dagwieers** [comment]
@romanvm The sources are coming directly from their websites

@basrieter Currently it is strictly Flemish (so not even Belgian), but I don't want to restrict it to just that.

File: plugin.video.regiotv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.regiotv" name="Regio TV" version="0.1.0" provider-name="dagwieers">
+<requires>
+  <import addon="script.module.routing" version="0.2.3"/>
+  <import addon="xbmc.python" version="2.25.0" />
```
Comment (Rechi) [inline plugin.video.regiotv/addon.xml]: > WARN: For leia it is advised to set xbmc.python version to 2.26.0

File: plugin.video.regiotv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    Beschikbare zenders: ATV, BRUZZ, Focus TV, ROB-tv, TVL, TV Oost, WTV
+
+    [I]Deze add-on wordt niet ondersteund door een televisiezender, en wordt aangeboden 'as is', zonder enige garantie.[/I]
+  </description>
+  <disclaimer lang="nl_NL">Deze add-on wordt niet ondersteund door een televisiezender, en wordt aangeboden 'as is', zonder enige garantie.</disclaimer>
+  <language>en nl</language>
+  <platform>all</platform>
+  <license>GNU General Public License, v3</license>
+  <website>https://github.com/dagwieers/plugin.video.regiotv/wiki</website>
```
Comment (Rechi) [inline plugin.video.regiotv/addon.xml]: > WARN: https://github.com/dagwieers/plugin.video.regiotv/wiki redirects to https://github.com/dagwieers/plugin.video.regiotv

File: plugin.video.regiotv/resources/lib/addon.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            mediatype='video',
+            playcount=0,
+            plot=plot,
+        ))
+        list_item.setArt(dict(
+            icon='DefaultAddonPVRClient.png',
+            thumb=channel.get('logo'),
+        ))
+
+        # FIXME: The setIsFolder method is new in Kodi18, so we cannot use it just yet.
```
Comment (Rechi) [inline plugin.video.regiotv/resources/lib/addon.py]: This add-on is submitted to the Leia branch, so you could use it.

File: plugin.video.regiotv/addon.xml (hunk 1 above)
Comment (dagwieers) [reply plugin.video.regiotv/addon.xml]: But we also would like to support Krypton. And not manage 2 versions.

File: plugin.video.regiotv/addon.xml (hunk 2 above)
Comment (dagwieers) [reply plugin.video.regiotv/addon.xml]: But we also would like to support Krypton. And not manage 2 versions.

File: plugin.video.regiotv/resources/lib/addon.py (hunk 3 above)
Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/addon.py]: But we also would like to support Krypton. And not manage 2 versions.

File: plugin.video.regiotv/addon.xml (hunk 2 above)
Comment (Rechi) [reply plugin.video.regiotv/addon.xml]: Please read my above comment.

Comment (dagwieers) [reply plugin.video.regiotv/addon.xml]: The wiki page will come, so that's fine.

File: plugin.video.regiotv/resources/lib/addon.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+    xbmcplugin.setContent(plugin.handle, content='files')
+    xbmcplugin.setPluginCategory(plugin.handle, category='Zenders')
+    xbmcplugin.setPluginFanart(plugin.handle, image='DefaultAddonPVRClient.png')
+    xbmcplugin.addSortMethod(plugin.handle, sortMethod=xbmcplugin.SORT_METHOD_UNSORTED)
+    xbmcplugin.addSortMethod(plugin.handle, sortMethod=xbmcplugin.SORT_METHOD_LABEL)
+
+    listing = []
+    for channel in CHANNELS:
+        is_folder = False
```
Comment (basrieter) [inline plugin.video.regiotv/resources/lib/addon.py]: the `is_folder` and `is_playable` values never change. So why all the checks on the values?

Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/addon.py]: They don't change at the moment, but they may in the future.
The code resembles other code, and I find it useful.

File: plugin.video.regiotv/addon.xml (hunk 2 above)
Comment (dagwieers) [reply plugin.video.regiotv/addon.xml]: The wiki page now exists.

**Comment by basrieter** [comment]
They are legit as far as I can see.

**Pull request merged**